relatedartist.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/zmb3/spotify"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+type relatedartist struct {
+	id         string
+	name       string
+	popularity int
```

Comment:
Do you need the popularity?

Reply:
not for now, but the plan is to use popularity in the algorithm, i.e. find related tracks with similar popularity and apply some logic to return a list.